__tests__/IRSReport.js

```diff
@@ -1,10 +1,20 @@
 jest.dontMock('../src/js/IRSReport.jsx');
+jest.dontMock('../src/js/api.js');
 
 var React = require('react');
 var ReactDOM = require('react-dom');
 var TestUtils = require('react-addons-test-utils');
 
+var superagent = require('superagent');
+var fs = require('fs');
+
 var IRSReport = require('../src/js/IRSReport.jsx');
+var irsJSON = fs.readFileSync('./server/json/irs.json');
+
+superagent.get = jest.genMockFn().mockReturnThis();
```

Comment:
Yes, I took the easy way out here ... for now. Want to move on to Signature and then Q029 and Q595. 
Once I have a better idea of whats going on here I will come back.